pkg/registry/server/server.go

```diff
@@ -233,3 +239,125 @@ func ResolveLiteralConfig(resolver Resolver, resolverMetrics *metrics.Metrics) h
 		resolveAndRespond(resolver, unresolvedConfig, w, logger, resolverMetrics)
 	}
 }
+
+func ResolveAndMergeConfigsAndInjectTest(configs Getter, resolver Resolver, resolverMetrics *metrics.Metrics) http.HandlerFunc {
```

Comment:
Let me look into that and see if it is necessary. I hadn't determined that we needed that one during my research, but I may have just missed it.

Reply:
I did find 2 usages of `operator` among components that contribute to the release payload. This does look like it will need to be merged as well. I will have an update shortly.

Reply:
It is possible we will end up with conflicts in the case that those exact 2 configs get merged, but we can fix that if it comes up. For now, it seems like overkill so I just simply merged them.